ads/_config.js

```diff
@@ -903,6 +903,15 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'strossle': {
+    preconnect: [
+      'https://amp.spklw.com',
+      'https://widgets.sprinklecontent.com',
+      'https://images.sprinklecontent.com',
+    ],
+    renderStartImplemented: true,
```

Comment:
So I went ahead and tested your `renderStart` implementation:

<img width="1379" alt="Screen Shot 2019-06-03 at 5 07 17 PM" src="https://user-images.githubusercontent.com/1448289/58812459-3a802f00-8622-11e9-9584-a33d6009ea73.png">

However, my `console.log` inside of the `amp-ad-xorigin-iframe-handler.js` is not called:

<img width="619" alt="Screen Shot 2019-06-03 at 5 08 57 PM" src="https://user-images.githubusercontent.com/1448289/58812520-5683d080-8622-11e9-854f-8cc6d2033f40.png">

Thus, I just wanted to double check if your server supports `renderStartImplemented`? If possible, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains. If not, could you please remove this line?

Let me know what you think. Thank you! 😄

Reply:
@torch2424 Sure, thanks showing me that line. 
For now we will skip this feature `renderStartImplemented `.

Looks like ok for now after removing that.
Could you please confirm that too?  😄